presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreOperationType.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+public enum MetastoreOperationType
+{
+    ADD_PARTITION,
+    ALTER_TABLES,
+    ALTER_PARTITIONS,
+    APPEND_PARTITION,
+    APPEND_PARTITION_BY_NAME,
+    CREATE_TABLE,
+    GET_PARTITION,
+    GET_PARTITION_BY_NAME,
+    GET_TABLE,
+    OTHERS
+}
```

Comment:
Same as above. Didn't find where this enum is used

Reply:
Same as the above, we used them in prismhivemetastore.